build.gradle

```diff
@@ -26,17 +25,19 @@ versionFile.mkdirs()
 versionFile.delete()
 versionFile << version
 
+final def internalRepoHost = System.getenv('SNPS_INTERNAL_ARTIFACTORY')
+
 repositories {
+    maven { url "${internalRepoHost}/artifactory/jcenter" }
     maven { url "https://sig-repo.synopsys.com/bds-bdio-release" }
 }
 
 dependencies {
     api 'com.synopsys.integration:blackduck-common-api:2023.4.2.5'
-    api 'com.synopsys.integration:phone-home-client:5.1.13'
+    api 'com.synopsys.integration:phone-home-client:6.0.0-SNAPSHOT'
```

Comment:
can we update this when the phone home client MR is merged so that it isn't a snapshot?

Reply:
Yup thanks, I'll make it a point to double check all versions before merging. Keeping this as a snapshot for now till we release phone-home-client.